libs/src/services/creatorNode/CreatorNode.ts

```diff
@@ -335,7 +335,7 @@ export class CreatorNode {
     trackFile: File,
```

Comment:
same here - I'll handle these TODOS as a followup and then either I or @sliptype can move them from libs to SDK